nemo/backends/pytorch/module_wrapper.py

```diff
@@ -40,8 +40,13 @@ def eval(self):
     def train(self):
         return self._pt_module.train()
 
-    def __call__(self, force_pt=False, *input, **kwargs):
-        pt_call = len(input) > 0 or force_pt
+    def __call__(self, force_pt: bool = False, *input, **kwargs):
+        pt_call = len(input) > 0 or force_pt if isinstance(force_pt, bool) else force_pt is not None
+        if not isinstance(force_pt, bool):
```

Comment:
not sure why this is is necessary. looks like a workaround some bug?

Reply:
basically, if force_pt isn't bool, it means it was called as a NeuralModule  but with args, not kwargs (which is likely a bug).

Reply:
There is a Torch call **passing a Tensor** to force_pt. We have to do handle this use case